epub33/core/index.html

```diff
@@ -5746,11 +5746,12 @@ <h4>File Paths and File Names</h4>
 				<section id="sec-file-names-to-path-names">
 					<h4>Deriving File Paths of Files</h4>
```

Comment:
Can "of Files" be stripped here? It's kind of redundant with "File Paths" and also doesn't account for directories.

Reply:
Right. Done.